test/integration/simple_project/inspec.yml

```
@@ -2,7 +2,7 @@ name: simple_project
 depends:
   - name: inspec-gcp
     git: https://github.com/inspec/inspec-gcp.git
-    version: ~> 0.9.0
+    version: ~> 0.11.0
```

It turns out that the git profile fetcher does not actually support the `version` property: https://github.com/inspec/inspec/issues/2986.

The work around is to use `tag`.

```suggestion
    tag: v0.11.0
```

---

Hmmm ok, the change made the tests work, that's all I know. I'll try with tag.